ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,278 @@
+      echo -e '[wazuh_pre-release]\ngpgcheck=1\ngpgkey=https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages-dev.wazuh.com/pre-release/yum/\nprotect=1' | tee /etc/yum.repos.d/wazuh.repo $debug
+    fi
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Wazuh repository could not be added"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Wazuh manager
+installWazuh() {
+
+    logger "Installing the Wazuh manager..."
+    $sys_type install wazuh-manager-${WAZUH_VERSION} -y -q 
+    if [ "$?" != 0 ]; then
+        echo "Error: Wazuh installation failed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Elasticsearch
+installElasticsearch() {
+
+    logger "Installing Open Distro for Elasticsearch..."
+    $sys_type install opendistroforelasticsearch-${OPENDISTRO_VERSION} -y -q 
+
+    if [ "$?" != 0 ]; then
+        echo "Error: Elasticsearch installation failed"
+        exit 1;
+    else
+        logger "Done"
+
+        logger "Configuring Elasticsearch..."
+
+
+        curl -so /etc/elasticsearch/elasticsearch.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/7.x/elasticsearch_all_in_one.yml --max-time 300
+        
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/roles.yml --max-time 300
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/roles_mapping.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/roles_mapping.yml --max-time 300
+        curl -so /usr/share/elasticsearch/plugins/opendistro_security/securityconfig/internal_users.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/elasticsearch/roles/internal_users.yml --max-time 300
+        rm /etc/elasticsearch/esnode-key.pem /etc/elasticsearch/esnode.pem /etc/elasticsearch/kirk-key.pem /etc/elasticsearch/kirk.pem /etc/elasticsearch/root-ca.pem -f
+        mkdir -p /etc/elasticsearch/certs $debug
+        cd /etc/elasticsearch/certs 
+        curl -so /etc/elasticsearch/certs/search-guard-tlstool-1.8.zip https://maven.search-guard.com/search-guard-tlstool/1.8/search-guard-tlstool-1.8.zip --max-time 300 
+        unzip search-guard-tlstool-1.8.zip -d searchguard 
+        curl -so /etc/elasticsearch/certs/searchguard/search-guard.yml https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}/resources/open-distro/searchguard/search-guard-aio.yml --max-time 300 
+        chmod +x searchguard/tools/sgtlstool.sh 
+        ./searchguard/tools/sgtlstool.sh -c ./searchguard/search-guard.yml -ca -crt -t /etc/elasticsearch/certs/ $debug 
+
+        if [ "$?" != 0 ]; then
+            echo "Error: certificates were not created"
```

Comment:
Use `logger` function.

Reply:
Done in 53208f3d9e390ee69e4f23799af5b3009001d9c2